src/Sylius/Behat/Page/Admin/PaymentMethod/CreatePageInterface.php

```diff
@@ -38,6 +38,11 @@ public function specifyCode($code);
     public function chooseGateway($gateway);
 
     /**
+     * @param string $channel
+     */
+    public function checkChannel($channel);
```

Comment:
`$channelName` (to be consistent with [this line](https://github.com/Sylius/Sylius/pull/6731/files#diff-6efacfc080b07be504ece5120657b557R55))